config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   root "welcome#index"
+  resources :foods
```

Comment:
Would have liked to see the `only` syntax used in conjunction with this view, so we aren't creating unused routes.